www/css/cytube.css

```diff
@@ -239,8 +241,9 @@ li.ui-sortable-helper, li.ui-sortable-placeholder + li.queue_entry {
 .server-msg-disconnect {
     border: 1px solid #ff0000;
     padding-top: 5px;
-    padding-bottom: 5px;
-    margin: 5px;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    height: 30px;
```

Comment:
The margin-top & margin-bottom are used to keep a gap between the chat and top of message buffer. Without the height of 30px the box height is shrunk from its correct size: 
![image](https://cloud.githubusercontent.com/assets/1368224/6451383/17350696-c0fc-11e4-8877-12895e7cf85f.png).